docs/sql-ref-ansi-compliance.md

```diff
@@ -407,6 +407,7 @@ Below is a list of all the keywords in Spark SQL.
 |DATEADD|non-reserved|non-reserved|non-reserved|
 |DATEDIFF|non-reserved|non-reserved|non-reserved|
 |DAY|non-reserved|non-reserved|non-reserved|
+|DAYS|non-reserved|non-reserved|non-reserved|
```

Comment:
As far as I remember we don't support such plural units in other functions of interval parsing. How about to not mix this new feature (new units) with making the parser stricter?

Reply:
it's not a new feature. `InternalUtils.stringToInterval` allows ending `s` in unit names.